components/frontend/src/header_footer/Footer.css

```diff
@@ -1,11 +1,11 @@
-@media print {
-    div.footer {
-        display: none;
-    }
+.MuiDivider-root {
+    padding: 20px;
```

Comment:
Looks fine on the sample images, but don't you want a difference between horizontal and vertical padding for responsive views?

Reply:
It seems to work fine for smaller widths. Of course, really small widths don't work well because we don't have breakpoints. Moreover, if you make the window narrower, the menu bar gets cut off before the footer doesn't have enough space.